tests/test_duckdb.py

```diff
@@ -71,4 +73,54 @@ def test_duckdb_from_iso8601_timestamp(timestamp, expected):
         .execute(f"select from_iso8601_timestamp('{timestamp}')")
         .fetchone()[0]
     )
-    assert parsed, expected
+    assert parsed == expected
```

Comment:
AHEM.  :sob: 

Fixing this bad assert (point in favor of assertEqual I suppose...) revealed that duckdb's version that I wrote is returning naive timestamps - which I think I actually did manually change from aware timestamps at one point... but Athena looks like it uses aware ones too....  Needs more investigation. I just fixed the tests to work here, but didn't change any business logic for now. 

Added a TODO in the code though.